ngmix/joint_prior.py

```diff
@@ -977,3 +988,44 @@ def fill_fdiff(self, pars, fdiff):
         fdiff[0:index] = sqrt(chi2)
 
         return index
+
+    def sample(self, n=None):
```

Comment:
The keyword `n` should be `nrand` here to match the other priors.

Reply:
We should make this uniform everywhere.  What should it be?  `n` or `nrand` or `size` ?

Reply:
I have been moving all functions to `nrand`.